src/createEslintConfig.ts

```diff
@@ -14,7 +14,7 @@ export async function createEslintConfig({
   pkg,
   rootDir,
   writeFile,
-}: CreateEslintConfigArgs): Promise<CLIEngine.Options['baseConfig']> {
+}: CreateEslintConfigArgs): Promise<CLIEngine.Options['baseConfig'] | void> {
```

Comment:
I typed it as `void` as well because it seems intentional that when `--writeFile` succeeds, it doesn't `return config` (because it does `return config` when it errors/the `catch` case), but I'm not sure if that actually is intentional. Maybe it makes sense to `return config` in all cases?